pom.xml

```diff
@@ -5,8 +5,8 @@
   <modelVersion>4.0.0</modelVersion>
 
   <groupId>com.sap.sgs.phosphor.fosstars</groupId>
-  <artifactId>fosstars-model</artifactId>
```

Comment:
As discussed with @mibo , how about haveing the groupId com.sap.phosphor.

Reply:
Yup.

Reply:
👍